### PR TITLE
new ToString for CsvOutputFormat

### PR DESCRIPTION
**Pull request by skunert**

### CODE REVIEW

File: stratosphere-java/src/main/java/eu/stratosphere/api/java/io/CsvOutputFormat.java [hunk 1]
```diff
@@ -200,8 +200,14 @@ public void writeRecord(T element) throws IOException {
...
-	
-	/**
+
+
+	@Override
+	public String toString() {
+		return "Csv Output (Path: " + this.getOutputFilePath() + "; Delimiter: " + this.fieldDelimiter + ")";
+	}
+
+    /**
```
Comment (uce) [inline stratosphere-java/src/main/java/eu/stratosphere/api/java/io/CsvOutputFormat.java:208]: Indentation is off

File: stratosphere-java/src/main/java/eu/stratosphere/api/java/io/CsvOutputFormat.java [hunk 2]
```diff
@@ -200,8 +200,14 @@ public void writeRecord(T element) throws IOException {
 	}
 	
 	// --------------------------------------------------------------------------------------------
-	
-	/**
+
```
Comment (uce) [inline stratosphere-java/src/main/java/eu/stratosphere/api/java/io/CsvOutputFormat.java]: Two white lines

**Comment by uce** [comment]
You should squash your commits into one before we can merge this.

**Comment by skunert** [comment]
done

**Comment by uce** [comment]
Merged in 363ea248a756c10e03544dd4664b7bb2ddde531f.

**Pull request closed without merging**